diff --git "a/.\\Maya2018\\include\\maya\\MPxCommand.h" "b/.\\MayaLT2018\\include\\maya\\MPxCommand.h"
index ce9bb67..aa97bfc 100644
--- "a/.\\Maya2018\\include\\maya\\MPxCommand.h"
+++ "b/.\\MayaLT2018\\include\\maya\\MPxCommand.h"
@@ -68,6 +68,9 @@ class OPENMAYA_EXPORT MPxCommand
 public:
 						MPxCommand();
 	virtual 			~MPxCommand();
+#if MAYA_LT
+    virtual void        deleteSurplusMemory();
+#endif
 	virtual MStatus   	doIt( const MArgList& args );
 	virtual MStatus   	undoIt( );
 	virtual MStatus   	redoIt( );
