diff --git "a/.\\Maya2018\\include\\maya\\MItDependencyGraph.h" "b/.\\MayaLT2018\\include\\maya\\MItDependencyGraph.h"
index 940f698..b197a98 100644
--- "a/.\\Maya2018\\include\\maya\\MItDependencyGraph.h"
+++ "b/.\\MayaLT2018\\include\\maya\\MItDependencyGraph.h"
@@ -145,6 +145,9 @@ public:
 									Traversal traversal = kDepthFirst,
 									Level level = kNodeLevel,
 									MStatus* ReturnStatus = NULL );
+#if MAYA_LT
+    virtual
+#endif
 	            ~MItDependencyGraph ();
 	MStatus		reset();
 	MStatus		resetTo( MObject& rootNode,
